ast/compile.go

```diff
@@ -1452,6 +1452,13 @@ func (c *Compiler) resolveAllRefs() {
 	}
 }
 
+func (c *Compiler) removeImports() {
+	for _, name := range c.sorted {
+		mod := c.Modules[name]
+		mod.Imports = nil
+	}
```

Comment:
[nit]
```suggestion
	for name := range c.Modules {
		c.Modules[name].Imports = nil
	}
```
...or something like that -- order doesn't matter here